MMC/SD/SDIO card support
This selects MultiMediaCard, Secure Digital and Secure
Digital I/O support.

If you want MMC/SD/SDIO support, you should say Y here and
also to your specific host controller driver.
Depends: HAS_IOMEM
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:20:24.334610312 -0400
@@ -1981,7 +1981,6 @@
 CONFIG_BT_HCIBTUSB_BCM=y
 CONFIG_BT_HCIBTUSB_MTK=y
 CONFIG_BT_HCIBTUSB_RTL=y
-CONFIG_BT_HCIBTSDIO=m
 CONFIG_BT_HCIUART=m
 CONFIG_BT_HCIUART_SERDEV=y
 CONFIG_BT_HCIUART_H4=y
@@ -2004,9 +2003,7 @@
 CONFIG_BT_HCIBLUECARD=m
 CONFIG_BT_HCIVHCI=m
 CONFIG_BT_MRVL=m
-CONFIG_BT_MRVL_SDIO=m
 CONFIG_BT_ATH3K=m
-CONFIG_BT_MTKSDIO=m
 CONFIG_BT_MTKUART=m
 CONFIG_BT_HCIRSI=m
 CONFIG_BT_VIRTIO=m
@@ -3574,7 +3571,6 @@
 CONFIG_CARL9170_WPC=y
 # CONFIG_CARL9170_HWRNG is not set
 CONFIG_ATH6KL=m
-CONFIG_ATH6KL_SDIO=m
 CONFIG_ATH6KL_USB=m
 CONFIG_ATH6KL_DEBUG=y
 # CONFIG_ATH6KL_TRACING is not set
@@ -3586,7 +3582,6 @@
 CONFIG_ATH10K=m
 CONFIG_ATH10K_CE=y
 CONFIG_ATH10K_PCI=m
-CONFIG_ATH10K_SDIO=m
 CONFIG_ATH10K_USB=m
 # CONFIG_ATH10K_DEBUG is not set
 CONFIG_ATH10K_DEBUGFS=y
@@ -3610,7 +3605,6 @@
 # CONFIG_B43_BUSES_SSB is not set
 CONFIG_B43_PCI_AUTOSELECT=y
 CONFIG_B43_PCICORE_AUTOSELECT=y
-CONFIG_B43_SDIO=y
 CONFIG_B43_BCMA_PIO=y
 CONFIG_B43_PIO=y
 CONFIG_B43_PHY_G=y
@@ -3637,7 +3631,6 @@
 CONFIG_BRCMFMAC=m
 CONFIG_BRCMFMAC_PROTO_BCDC=y
 CONFIG_BRCMFMAC_PROTO_MSGBUF=y
-CONFIG_BRCMFMAC_SDIO=y
 CONFIG_BRCMFMAC_USB=y
 CONFIG_BRCMFMAC_PCIE=y
 # CONFIG_BRCM_TRACING is not set
@@ -3692,13 +3685,11 @@
 CONFIG_LIBERTAS=m
 CONFIG_LIBERTAS_USB=m
 CONFIG_LIBERTAS_CS=m
-CONFIG_LIBERTAS_SDIO=m
 # CONFIG_LIBERTAS_SPI is not set
 # CONFIG_LIBERTAS_DEBUG is not set
 CONFIG_LIBERTAS_MESH=y
 # CONFIG_LIBERTAS_THINFIRM is not set
 CONFIG_MWIFIEX=m
-CONFIG_MWIFIEX_SDIO=m
 CONFIG_MWIFIEX_PCIE=m
 CONFIG_MWIFIEX_USB=m
 CONFIG_MWL8K=m
@@ -3707,7 +3698,6 @@
 CONFIG_MT76_CORE=m
 CONFIG_MT76_LEDS=y
 CONFIG_MT76_USB=m
-CONFIG_MT76_SDIO=m
 CONFIG_MT76x02_LIB=m
 CONFIG_MT76x02_USB=m
 CONFIG_MT76_CONNAC_LIB=m
@@ -3722,14 +3712,11 @@
 CONFIG_MT7615E=m
 CONFIG_MT7663_USB_SDIO_COMMON=m
 CONFIG_MT7663U=m
-CONFIG_MT7663S=m
 CONFIG_MT7915E=m
 CONFIG_MT7921_COMMON=m
 CONFIG_MT7921E=m
-CONFIG_MT7921S=m
 CONFIG_MT7921U=m
 CONFIG_WLAN_VENDOR_MICROCHIP=y
-# CONFIG_WILC1000_SDIO is not set
 # CONFIG_WILC1000_SPI is not set
 # CONFIG_WLAN_VENDOR_PURELIFI is not set
 CONFIG_WLAN_VENDOR_RALINK=y
@@ -3810,23 +3797,18 @@
 CONFIG_WLAN_VENDOR_RSI=y
 CONFIG_RSI_91X=m
 CONFIG_RSI_DEBUGFS=y
-CONFIG_RSI_SDIO=m
 CONFIG_RSI_USB=m
 CONFIG_RSI_COEX=y
 # CONFIG_WLAN_VENDOR_SILABS is not set
 CONFIG_WLAN_VENDOR_ST=y
 CONFIG_CW1200=m
-CONFIG_CW1200_WLAN_SDIO=m
 CONFIG_CW1200_WLAN_SPI=m
 CONFIG_WLAN_VENDOR_TI=y
 CONFIG_WL1251=m
 CONFIG_WL1251_SPI=m
-CONFIG_WL1251_SDIO=m
 CONFIG_WL12XX=m
 CONFIG_WL18XX=m
 CONFIG_WLCORE=m
-CONFIG_WLCORE_SDIO=m
-CONFIG_WILINK_PLATFORM_DATA=y
 CONFIG_WLAN_VENDOR_ZYDAS=y
 # CONFIG_USB_ZD1201 is not set
 CONFIG_ZD1211RW=m
@@ -5084,8 +5066,6 @@
 CONFIG_SSB_B43_PCI_BRIDGE=y
 CONFIG_SSB_PCMCIAHOST_POSSIBLE=y
 CONFIG_SSB_PCMCIAHOST=y
-CONFIG_SSB_SDIOHOST_POSSIBLE=y
-CONFIG_SSB_SDIOHOST=y
 CONFIG_SSB_DRIVER_PCICORE_POSSIBLE=y
 CONFIG_SSB_DRIVER_PCICORE=y
 CONFIG_SSB_DRIVER_GPIO=y
@@ -5722,11 +5702,6 @@
 #
 
 #
-# MMC/SDIO DVB adapters
-#
-CONFIG_SMS_SDIO_DRV=m
-
-#
 # FireWire (IEEE 1394) Adapters
 #
 CONFIG_DVB_FIREDTV=m
@@ -5745,7 +5720,6 @@
 CONFIG_VIDEO_SAA7146_VV=m
 CONFIG_SMS_SIANO_MDTV=m
 CONFIG_SMS_SIANO_RC=y
-# CONFIG_SMS_SIANO_DEBUGFS is not set
 CONFIG_VIDEOBUF2_CORE=m
 CONFIG_VIDEOBUF2_V4L2=m
 CONFIG_VIDEOBUF2_MEMOPS=m
@@ -7422,41 +7396,7 @@
 
 CONFIG_USB_ROLE_SWITCH=y
 CONFIG_USB_ROLES_INTEL_XHCI=m
-CONFIG_MMC=m
-CONFIG_MMC_BLOCK=m
-CONFIG_MMC_BLOCK_MINORS=8
-CONFIG_SDIO_UART=m
-# CONFIG_MMC_TEST is not set
-# CONFIG_MMC_CRYPTO is not set
-
-#
-# MMC/SD/SDIO Host Controller Drivers
-#
-# CONFIG_MMC_DEBUG is not set
-CONFIG_MMC_SDHCI=m
-CONFIG_MMC_SDHCI_IO_ACCESSORS=y
-CONFIG_MMC_SDHCI_PCI=m
-CONFIG_MMC_RICOH_MMC=y
-CONFIG_MMC_SDHCI_ACPI=m
-CONFIG_MMC_SDHCI_PLTFM=m
-# CONFIG_MMC_SDHCI_F_SDH30 is not set
-CONFIG_MMC_WBSD=m
-CONFIG_MMC_ALCOR=m
-CONFIG_MMC_TIFM_SD=m
-# CONFIG_MMC_SPI is not set
-CONFIG_MMC_SDRICOH_CS=m
-CONFIG_MMC_CB710=m
-CONFIG_MMC_VIA_SDMMC=m
-CONFIG_MMC_VUB300=m
-CONFIG_MMC_USHC=m
-# CONFIG_MMC_USDHI6ROL0 is not set
-CONFIG_MMC_REALTEK_PCI=m
-CONFIG_MMC_REALTEK_USB=m
-CONFIG_MMC_CQHCI=m
-CONFIG_MMC_HSQ=m
-CONFIG_MMC_TOSHIBA_PCI=m
-# CONFIG_MMC_MTK is not set
-CONFIG_MMC_SDHCI_XENON=m
+# CONFIG_MMC is not set
 # CONFIG_SCSI_UFSHCD is not set
 CONFIG_MEMSTICK=m
 # CONFIG_MEMSTICK_DEBUG is not set
@@ -7959,7 +7899,6 @@
 CONFIG_RTLLIB_CRYPTO_TKIP=m
 CONFIG_RTLLIB_CRYPTO_WEP=m
 CONFIG_RTL8192E=m
-CONFIG_RTL8723BS=m
 CONFIG_R8712U=m
 CONFIG_R8188EU=m
 # CONFIG_RTS5208 is not set
@@ -8030,7 +7969,6 @@
 # CONFIG_LTE_GDM724X is not set
 # CONFIG_FIREWIRE_SERIAL is not set
 # CONFIG_FB_TFT is not set
-# CONFIG_KS7010 is not set
 # CONFIG_PI433 is not set
 # CONFIG_FIELDBUS_DEV is not set
 CONFIG_QLGE=m
